bindgen/templates/ObjectTemplate.cs

```diff
@@ -7,7 +7,15 @@
 {%- if self.include_once_check("ObjectRuntime.cs") %}{% include "ObjectRuntime.cs" %}{% endif %}
 
 {%- call cs::docstring(obj, 0) %}
-{{ config.access_modifier() }} interface I{{ type_name }} {
+{{ config.access_modifier() }} interface I{{ type_name }}
+    {%- for tm in obj.uniffi_traits() -%}
+    {%- if loop.first -%} : {% endif -%}
```

Comment:
Seems like the colon should go inside `UniffiTrait::Eq` branch. If object had only implemented `Hash` trait without `Eq`, the colon would be inserted without leading `IEquatable`.